MapboxDirectionsTests/RoutableMatch.swift

```diff
@@ -0,0 +1,111 @@
+import XCTest
+import OHHTTPStubs
+@testable import MapboxDirections
+
+class RoutableMatchTest: XCTestCase {
+    override func tearDown() {
+        OHHTTPStubs.removeAllStubs()
+        super.tearDown()
+    }
+    
+    func test() {
```

Comment:
Rename this method to something more descriptive. Consider moving this method into either MatchTests or V5Tests.